mate-panel/applet.c

```diff
@@ -363,6 +363,12 @@ setup_an_item (AppletUserMenu *menu,
 	       GtkWidget      *submenu,
 	       int             is_submenu)
 {
+#if GTK_CHECK_VERSION(3, 10, 0)
+	/* Drop menu item image
+	 * See: https://docs.google.com/document/d/1KCVPoYQBqMbDP11tHPpjW6uaEHrvLUmcDPqKAppCY8o/pub
```

Comment:
fuck gnome devs, I like icons at menu :)
gtk_menu_item_new_with_mnemonic would be dropped only at gtk4 or so on because of backward compatibility.